subsys/bluetooth/host/gatt_internal.h

```diff
@@ -34,6 +34,11 @@ int bt_gatt_store_ccc(uint8_t id, const bt_addr_le_t *addr);
 
 int bt_gatt_clear(uint8_t id, const bt_addr_le_t *addr);
 
+void bt_gatt_update_ccc_cfg_addr(const bt_addr_le_t *private_addr, const bt_addr_le_t *id_addr);
+int bt_gatt_store_ccc_and_cf(struct bt_conn *conn);
+void bt_gatt_identity_resolved(struct bt_conn *conn, const bt_addr_le_t *private_addr,
+							   const bt_addr_le_t *id_addr);
+
```

Comment:
All of this should be removed, now that nothing is getting called from outside of `gatt.c`.

Reply:
Done.